src/screens/settings/SettingsReaderScreen/Modals/CustomFileModal.tsx

```diff
@@ -35,6 +35,7 @@ const CustomFileModal: React.FC<CustomCSSModalProps> = ({
   ),
   placeholder = '',
 }) => {
+  const [text, setText] = useState(customFile);
```

Comment:
Sorry, I missed this PR before. I have fixed the TextInput's behaviour in 14ed97b2c342895f0976f2d9734b97875e941a9e so this fix shouldn't be needed anymore.

Reply:
No problem, but I can't change anything for a week now since I'm on holiday.

Reply:
I removed your version and added mine, because your version didn't support the "Not saved" text and couldn't show the file text after opening from file.